### ISSUE TITLE
Trigger Action Monitor Action when ACF Field Groups are saved

### ISSUE THREAD

**Issue by jasonbahl** [gatsbyjs/wp-gatsby#70]
#69 fixes a bug where post_types that shouldn't have been tracked by Gatsby Action monitor are no longer tracked. The tradeoff is that post types that modify the Schema, such as ACF Fields, no longer trigger updates to Gatsby. 

We should trigger an action when ACF Field Groups are changed to notify Gatsby that the Schema needs to be re-fetched.

**Comment by jasonbahl** [comment]
@TylerBarnes what action would you like triggered here?

**Comment by TylerBarnes** [comment]
I think we should go with "DIFF_SCHEMAS" as the action type name but it actually doesn't matter what it is as long as the source plugin ignores the action.. which is kinda funny. Any action will cause the schemas to be diffed but the source plugin only responds to specific actions. In the case of an action for this, there's nothing additional the source plugin will need to do.

**Comment by jasonbahl** [comment]
Interesting. 🤔 

So we can potentially have something like: 

```
$this->trigger_schema_diff();
```

And that would store an action like: 

```
- **action_type**: DIFF_SCHEMAS
- **referenced_node_id**: none
- **referenced_node_type**: none
```

And any other actions that trigger that can just update the modified timestamp of that action. So then any event we detect that should cause a remote schema diff can trigger that action and we'll be good?

**Comment by TylerBarnes** [comment]
Yep :) exactly

**Comment by jasonbahl** [comment]
I have some ideas for WPGraphQL to do some Schema caching/diffing itself which would trigger an action such as `do_action( 'graphql_schema_changed', $new_schema_sdl, $old_schema_sdl )` and we could use this to trigger the action for Gatsby too. 

That will come later for WPGraphQL though as I need to put more thought into it.

**Comment by TylerBarnes** [comment]
ooo very nice :)

**Comment by TylerBarnes** [comment]
This was fixed in #74
